src/index.html

```diff
@@ -6,9 +6,81 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav>
+        <ul class="menu">
+          <li class="menu__item">
+            <a href="#apple"
+              class="is-active menu__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
+            <a href="#samsung"
+              class="menu__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu__item">
+            <a href="#смартфоны_и_телефоны"
```

Comment:
don't look at the text language that you render on the website. It can change. Better use English for code